### PR TITLE
LPD-41420 Frontend changes for product configuration eligibility admin management

### PR DESCRIPTION
**Pull request by dsitu**
https://liferay.atlassian.net/browse/LPD-41420

This is code related to the frontend part of product configuration eligibility admin management. This PR is built ontop of https://github.com/liferay-commerce/liferay-portal/pull/5701. Do not forward this until https://github.com/liferay-commerce/liferay-portal/pull/5701 is merged.

### CODE REVIEW

**Comment by dsitu** [comment]
ci:test:sf

**Comment by dsitu** [comment]
ci:test:sf

**Comment by dsitu** [comment]
ci:test:relevant

File: modules/apps/commerce/commerce-product-definitions-web/src/main/java/com/liferay/commerce/product/definitions/web/internal/display/context/CPConfigurationListQualifiersDisplayContext.java [hunk 1]
```diff
@@ -130,6 +130,35 @@ public CPConfigurationList getCPConfigurationList() throws PortalException {
...
+	}
+
+	public String getCPConfigurationListAccountGroupsAPIURL()
+		throws PortalException {
+
+		return StringBundler.concat(
+			"/o/headless-commerce-admin-catalog/v1.0",
+			"/product-configuration-lists/", getCPConfigurationListId(),
+			"/product-configuration-list-account-groups?nestedFields=",
+			"accountGroup");
```
Comment (andrea-ale-sbarra) [inline modules/apps/commerce/commerce-product-definitions-web/src/main/java/com/liferay/commerce/product/definitions/web/internal/display/context/CPConfigurationListQualifiersDisplayContext.java:184]: same here

File: modules/apps/commerce/commerce-product-definitions-web/src/main/java/com/liferay/commerce/product/definitions/web/internal/display/context/CPConfigurationListQualifiersDisplayContext.java [hunk 2]
```diff
@@ -168,6 +181,34 @@ public String getCPConfigurationListAccountsAPIURL()
...
+			false);
+	}
+
+	public String getCPConfigurationListChannelsAPIURL()
+		throws PortalException {
+
+		return StringBundler.concat(
+			"/o/headless-commerce-admin-catalog/v1.0",
+			"/product-configuration-lists/", getCPConfigurationListId(),
+			"/product-configuration-list-channels?nestedFields=channel");
```
Comment (andrea-ale-sbarra) [inline modules/apps/commerce/commerce-product-definitions-web/src/main/java/com/liferay/commerce/product/definitions/web/internal/display/context/CPConfigurationListQualifiersDisplayContext.java:221]: also here

File: modules/apps/commerce/commerce-product-definitions-web/src/main/java/com/liferay/commerce/product/definitions/web/internal/display/context/CPConfigurationListQualifiersDisplayContext.java [hunk 3]
```diff
@@ -209,6 +221,35 @@ public String getCPConfigurationListChannelsAPIURL()
...
+			false);
+	}
+
+	public String getCPConfigurationListOrderTypesAPIURL()
+		throws PortalException {
+
+		return StringBundler.concat(
+			"/o/headless-commerce-admin-catalog/v1.0",
+			"/product-configuration-lists/", getCPConfigurationListId(),
+			"/product-configuration-list-order-types?nestedFields=",
```
Comment (andrea-ale-sbarra) [inline modules/apps/commerce/commerce-product-definitions-web/src/main/java/com/liferay/commerce/product/definitions/web/internal/display/context/CPConfigurationListQualifiersDisplayContext.java:249]: also here

File: modules/apps/commerce/commerce-product-definitions-web/src/main/java/com/liferay/commerce/product/definitions/web/internal/frontend/taglib/servlet/taglib/CPConfigurationListQualifiersScreenNavigationEntry.java [hunk 4]
```diff
@@ -0,0 +1,96 @@
...
+	extends CPConfigurationListQualifiersScreenNavigationCategory
+	implements ScreenNavigationEntry<CPConfigurationList> {
+
+	@Override
+	public String getEntryKey() {
+		return getCategoryKey();
+	}
+
+	@Override
+	public void render(
```
Comment (andrea-ale-sbarra) [inline modules/apps/commerce/commerce-product-definitions-web/src/main/java/com/liferay/commerce/product/definitions/web/internal/frontend/taglib/servlet/taglib/CPConfigurationListQualifiersScreenNavigationEntry.java:47]: I've sent you a commit to make this entry visible only for list that are not master

**Review by andrea-ale-sbarra — CHANGES_REQUESTED** [review]
changes that are consequence of the change request on the headless layer + please merge the commit that I've sent you + please add the check also at the service layer so that the same result can be achieved with the headless request => eligibility can be added only for non master list

thank you!

**Comment by dsitu** [comment]
@andrea-ale-sbarra Added commit https://github.com/liferay-commerce/liferay-portal/pull/5702/commits/8e2a3837f3c5b6480ad448e2df1542c9ca6260ed

**Review by andrea-ale-sbarra — APPROVED** [review]
please have a look at my change requests in the other PR, this looked good at first review already, since you made the changes in the other PR I think this is good to go.

There's a rebase error.

Thank you!

**Comment by dsitu** [comment]
ci:test:sf

**Comment by dsitu** [comment]
ci:test:sf

**Comment by dsitu** [comment]
ci:test:relevant

**Comment by dsitu** [comment]
ci:test:sf

**Comment by dsitu** [comment]
ci:test:relevant

**Comment by dsitu** [comment]
Closing this as I split it up to separate PR's.
1. https://github.com/liferay-commerce/liferay-portal/pull/5745
2. https://github.com/liferay-commerce/liferay-portal/pull/5746

**Pull request closed without merging**